src/components/giving-tuesday/Countdown.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import CountdownPie from './CountdownPie';
+import UseCountdown from '../../hooks/useCountdown';
+
+const Countdown = () => {
+  const targetDate = new Date('November 28, 2023').toLocaleString('en-US', {
```

Comment:
@wmui51 Have you tested with people at different time zones? This looks like it should work as expected, but probably won't hurt to double check. I'm going to merge this in so you all can do the testing.